android-sdk/src/main/java/com/checkout/android_sdk/logging/FramesLoggingEventDataProvider.kt

```diff
@@ -1,7 +1,7 @@
 package com.checkout.android_sdk.logging
 import com.checkout.android_sdk.Utils.Environment
 import com.checkout.eventlogger.domain.model.MonitoringLevel
-import java.util.Locale
+import java.util.*
```

Comment:
do we need to make this change?

Reply:
When the studio's default shortcut formatting is applied. It automatically every time updating. I will change this.